pkg/provider/provider.go

```diff
@@ -78,10 +79,13 @@ type TestEnvironment struct { // rename this with testTarget
 	StatefulSets []*StatefulSet `json:"testStatefulSets"`
 
 	// Note: Containers is a filtered list of objects based on a block list of disallowed container names.
-	Containers             []*Container `json:"testContainers"`
-	Operators              []*Operator  `json:"testOperators"`
-	PersistentVolumes      []corev1.PersistentVolume
-	PersistentVolumeClaims []corev1.PersistentVolumeClaim
+	Containers               []*Container `json:"testContainers"`
+	Operators                []*Operator  `json:"testOperators"`
+	AllOperators             []*Operator  `json:"AllOperators"`
```

Comment:
I've tested this locally with the "-f observability" only and the claim.json file increased from 1.2MB (main branch) to 22MB (this PR's branch). I think we should create custom json serialization methods for this structures to avoid adding so much information into the claim file.

Reply:
Fixed. Selectively dump raw CSV only for operator under test.